src/reuse/_util.py

```diff
@@ -413,7 +426,12 @@ def _checksum(path: PathLike) -> str:
 class PathType:
     """Factory for creating Paths"""
 
-    def __init__(self, mode="r", force_file=False, force_directory=False):
+    def __init__(
+        self,
+        mode: str = "r",
```

Comment:
This type is probably incorrect

Reply:
Should be `Literal`, but only available in Python 3.8+.

Reply:
You could add a comment for future reference:
```suggestion
        mode: str = "r", # change to Literal for python3.8+
```

Reply:
I looked at this: there are too many possible permutations for me to bother with this. I'll just lazily keep it as-is.